components/authz-service/server/v2/policy.go

```diff
@@ -78,12 +78,15 @@ func NewPoliciesServer(
 	pl storage_v1.PoliciesLister,
 	vChan chan api.Version) (PolicyServer, error) {
 
+	policyRefersher := NewPolicyRefresher(ctx, l, s, e)
```

Comment:
[nit] I guess if this is only used within this package, we could _not export_ it.

Reply:
same spelling error-- but at least it's consistent!